source/class/qx/tool/cli/commands/Command.js

```diff
@@ -128,7 +128,7 @@ qx.Class.define("qx.tool.cli.commands.Command", {
      * Returns the path to the current library. If the current directory contains several libraries,
      * the first one found is returned.
      * @throws {Error} Throws an error if no library can be found.
-     * @return {Promise<String>)} A promise that resolves with the absolute path to the library
+     * @return {Promise} A promise that resolves with the absolute path to the library
```

Comment:
This is wrong, it should be just `@return {String}`

Reply:
Hm, an async function is always returning a Promise. I don't think that the JSDoc `@return` statement in async functions makes this implicit and states the return value only (since the Promise can also be rejected). Since the function is async, it should be documented to return a Promise, don't you think?

Reply:
AIUI there isnt an official jsdoc for `Promise<String>`, but IMHO the asynchronous nature of the function is what the `async` modifier is for; it is a language-level feature to describe that the function returns a promise, so we do not need to duplicate that in the jsdoc.  From the other point of view, if you have an async function, would it need to have it's `@return` documented as a Promise?  I think that would be a bit unnatural, because the async/await keywords are there to blend Promises into the background

Whether a promise is rejected or an exception can be thrown is not documented by `@return` either, so I don't think that's relevant (i think that there is a `@throws` available but its not used in the framework)